docs/deploy/setup-iomesh.md

```diff
@@ -84,11 +84,14 @@ spec:
 ```
 
 #### Mount Type
+In `hybrid-flash` deployment mode, IOMesh provides 2 mount types:
 
-IOMesh provides 2 mount types now:
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
+- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommended.
 
-- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
-- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
+In `all-flash` deployment mode, IOMesh only provides 1 mount type:
+
+- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```

Comment:
- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `dataStore`. Either `SATA` or `NVMe` SSD is recommended.